server/src/api/index.ts

```diff
@@ -0,0 +1,34 @@
+import * as dotenv from 'dotenv';
+import * as express from 'express';
```

Comment:
These imports are also weird, what's the difference between that and `import express from 'express'`?

Reply:
If I'm remembering right, it depends on how you set up you TSconfig to handle default exports that may or may not exist. I'll look at this again.

Reply:
After moving to esModuleInterop to true, I went through and fixed the ones I found that did this.